calendar-be/PlannerApp/src/main/java/com/PlannerApp/PlannerApp/Controller/InviteController.java

```diff
@@ -0,0 +1,26 @@
+package com.PlannerApp.PlannerApp.Controller;
+
+import com.PlannerApp.PlannerApp.Models.Invite;
+import com.PlannerApp.PlannerApp.Models.User;
+import com.PlannerApp.PlannerApp.Services.InviteService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/invite")
+@RequiredArgsConstructor
+@Slf4j
+public class InviteController {
+    private final InviteService inviteService;
+
+    @PostMapping("/insert")
+    @ResponseStatus(HttpStatus.CREATED)
+    public UUID insertGroup(@RequestBody Invite invite){
```

Comment:
Why is the name insertGroup?

Reply:
Because it insers the inviation to database.